pwndbg/wrappers/checksec.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import pwndbg.commands
+import pwndbg.wrappers
+
+
+cmd_name = "checksec"
+
+@pwndbg.memoize.reset_on_objfile
```

Comment:
These three decorators are used repeatedly throughout the module.  Might make sense to consolidate them to one decorator.